dot/core/syncer.go

```diff
@@ -125,19 +125,28 @@ func NewSyncer(cfg *SyncerConfig) (*Syncer, error) {
 
 // Start begins the syncer
 func (s *Syncer) Start() {
+	// thread safe change stopped to 1
+	canLock := atomic.CompareAndSwapUint32(&s.stopped, 0, 1)
+	if !canLock {
+		panic("[sync] Error when trying to change Syncer from stopped to started.")
```

Comment:
return error

Reply:
The start method does not return error right now. Are you sure this change is required ?

Reply:
yes, the syncer is only started in one place, so I don't think it's a lot of logic to add for an err check.

Reply:
also, you are chaning `s.stopped` to 1 in the start method, shouldn't it be changed to `0`?

Reply:
I updated this to started, please re-check.